packages/bsky/tests/views/__snapshots__/profile.test.ts.snap

```diff
@@ -469,3 +469,55 @@ Object {
   },
 }
 `;
+
+exports[`pds profile views status returns active status when within the duration 1`] = `
+Object {
+  "embed": Object {
+    "$type": "app.bsky.embed.external#external",
```

Comment:
This is supposed to be an `app.bsky.embed.external#view`.  I don't think that `app.bsky.embed.external#external` exists, actually!